Makefile

```diff
@@ -0,0 +1,40 @@
+SHELL=/bin/bash
+DATETIME:=$(shell date -u +%Y%m%dT%H%M%SZ)
+
+### Dependency commands ###
+
+install: ## Install dependencies and CLI app
+	pipenv install --dev
+
+update: install ## Update all Python dependencies
+	pipenv clean
+	pipenv update --dev
+
+### Test commands ###
+
+test: ## Run tests and print a coverage report
+	pipenv run coverage run --source=app -m pytest -vv
```

Comment:
I was actually going to propose adding `-vv` because I almost always need to use it for troubleshooting